sdk/appconfiguration/Azure.ResourceManager.AppConfiguration/tests/Tests/SnapshotCollectionTests.cs

```diff
@@ -0,0 +1,123 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Linq;
+using System.Threading.Tasks;
+using Azure.Core.TestFramework;
+using Azure.ResourceManager.AppConfiguration.Models;
+using Azure.ResourceManager.Resources;
+using NUnit.Framework;
+
+namespace Azure.ResourceManager.AppConfiguration.Tests
+{
+    public class SnapshotCollectionTests : AppConfigurationClientBase
```

Comment:
Please follow https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/resourcemanager/Azure.ResourceManager/docs/TestGuide.md to record the tests